assets/src/stories-editor/blocks/amp-story-cta/edit.css

```diff
@@ -5,10 +5,17 @@
 	height: 20%;
 }
 
-.editor-block-list__layout div[data-type="amp/amp-story-cta"] * {
+.block-editor-block-list__layout div[data-type="amp/amp-story-cta"]:not(#id) {
+	margin-top: 0;
+}
+
+.editor-block-list__layout div[data-type="amp/amp-story-cta"] img {
+
+	/* Remove the top/bottom padding from the available space. */
+	max-height: calc(110px - 24px);
 
-	/* 20% of the full height of a page in the editor */
-	max-height: 110px;
+	/* Counteract the height increase caused by the inline-block. */
+	margin-bottom: -9px;
```

Comment:
@mehigh Looks like the margin-bottom here is causing the vertical alignment display incorrectly in the editor, this can be seen in case of tiny images specifically, for example:
<img width="399" alt="Screenshot 2019-07-22 at 21 22 19" src="https://user-images.githubusercontent.com/3294597/61655236-d3e4cc80-acc6-11e9-869d-77b832ac36ca.png">

Without this margin:
<img width="409" alt="Screenshot 2019-07-22 at 21 22 27" src="https://user-images.githubusercontent.com/3294597/61655256-db0bda80-acc6-11e9-82f2-fe9681f38419.png">

Perhaps it's OK to remove this afterall?

Reply:
Yeah, let's remove it.